controls/os_spec.rb

```diff
@@ -228,3 +228,19 @@
     it { should be_empty }
   end
 end
+
+control 'os-10' do
+  impact 1.0
+  title "CIS: Disable unused filesystems"
+  desc "1.1.1 Ensure mounting of cramfs, freevxfs, jffs2, hfs, hfsplus, squashfs, udf, FAT"
+  describe file('/etc/modprobe.d/CIS.conf') do
```

Comment:
@bitvijays This tests verifies that all those filesystems need to be enabled. I think we can decide on best-practices for production environments, but I am not sure if we want to ensure all of those are activated.